src/main/scala/com/fulcrumgenomics/fasta/CollectAlternateContigNames.scala

```diff
@@ -112,6 +112,9 @@ object SequenceRole extends FgBioEnum[SequenceRole] {
     |
     |First, contig with the Sequence-Role "assembled-molecule" will be outputted.  Next, the remaining contigs will be
     |sorted by descending length.
+    |
+    |Aliases for values in the Assigned-Molecule column will only be applied to sequences with the "assembled-molecule"
```

Comment:
Two thoughts:
1. You could make this an option (with default) so it does the right thing for human-like assembly report.  I don't know enough about assembly reports for less finished organisms to know if we'd want to do something different there?  /shrug.
2. If you leave as is, maybe reword to "The `Assigned-Molecule` column, if specified as an `--alternate` will only be used for sequences with `Sequence-Role` `assembled-molecule`.